easybuild/easyconfigs/s/smfishHmrf/smfishHmrf-1.3.3-foss-2022a.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'smfishHmrf'
+version = '1.3.3'
+
+homepage = 'https://pypi.org/project/smfishHmrf'
+description = "smFish spatial pattern mining and cell type prediction"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['2fed45be8b5a5d47657cd4c8e9023b9ea6624863fc55a12afda092c6be37ca93']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Seaborn', '0.12.1'),
+    ('matplotlib', '3.5.2'),
+    ('scikit-learn', '1.1.2'),
+    ('Java', '11.0.16', '', SYSTEM),
```

We should use the `Java/11` wrapper here:

```suggestion
    ('Java', '11', '', SYSTEM),

---

I'll apply this change, so we can get this PR merged...